qcodes/dataset/sqlite/queries.py

```diff
@@ -829,10 +831,47 @@ def get_guid_from_run_id(conn: ConnectionPlus, run_id: int) -> str:
     Args:
         conn: database connection
         run_id: id of the run
+
+    Returns:
+        The guid of the run_id.
     """
     return select_one_where(conn, "runs", "guid", "run_id", run_id)
 
 
+def get_guid_from_multiple_run_ids(db_path: Union[str, Path],
```

Comment:
plural of guid, right?
```suggestion
def get_guids_from_multiple_run_ids(db_path: Union[str, Path],
```

Reply:
It can be guid if only one run provided, but yes, plural makes more sense :)